content/docs/command/command_line_part2.md

```diff
@@ -13,3 +13,181 @@ description = "Goals"
+| `pwd`   | print working directory (print the path to where I currently am in the file system)                                                                                                                                                    |
+| `cd`    | change directory (this moves you, the viewer of the terminal, to a new location in the file system). You may use an absolute file path or a relative path where `.` is the current working directory and `..` is the parent directory. |
+| `ls`    | list the contents of the directory                                                                                                                                                                                                     |
+| `mkdir` | make a directory                                                                                                                                                                                                                       |
+| `touch` | create a new file                                                                                                                                                                                                                      |
+| `rm`    | delete a file (note: to delete a directory, add the option `-r` or `-d`)                                                                                                                                                               |
+| `cp`    | copy a file (note: to copy a directory you probably need to add an option (like `-r`))                                                                                                                                                 |
+| `mv`    | move a file or directory to a new location (note: this can also be useful for changing the name of a file/directory)                                                                                                                   |
+| `echo`  | print some text                                                                                                                                                                                                                        |
+| `cat`   | print the contents of the file                                                                                                                                                                                                         |
+| `>`     | this is an operator: `X > Y` will redirect (send) the outputs of command `X` to `Y`; if `Y` exists, it this will _overwrite_ the existing file.                                                                                        |
+| `>>`    | this is an operator similar to the above: `X >> Y` will send the outputs of command `X` to `Y`; if `Y` exists, it this will _append_ to the existing file.                                                                             |
+
```

Comment:
I think we should point out that we can create a script file where we group some commands to achieve certain automation.

Reply:
will add a note but also mention this in the demo section